### PR TITLE
edit the readme

### PR DESCRIPTION
**Pull request by omrishuva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,14 @@ so you might want to do something like that:
 
 if you want the counter to persist to database once every 100 views.
 
+if you need to run callbacks for whatever reason after you update the counter just:
+  
+   class Foo < ActiveRecord::Base
+      include VisitCounter
+      cached_counter :counter_name
+      persist_with_callbacks = true
```
Comment (unorthodoxgeek) [inline README.md]: should be

``` ruby
self.persist_with_callbacks = true
```

Comment (unorthodoxgeek) [inline README.md]: was my mistake originally. please, if you're at it, update the same on the other example I have there...

**Comment by unorthodoxgeek** [comment]
muchas geracias!

**Comment by omrishuva** [comment]
de nada amigo

**Pull request merged**